registry/modules/rules_rust/0.22.0/source.json

```diff
@@ -1,5 +1,5 @@
 {
-  "url": "https://github.com/bazelbuild/rules_rust/releases/download/0.17.0/rules_rust-v0.17.0.tar.gz",
+  "url": "https://github.com/bazelbuild/rules_rust/releases/download/0.22.0/rules_rust-v0.22.0.tar.gz",
   "integrity": "sha384-v3eVoRi4rdkELxzV3egjTJkc8M3UH1YmbdZ2m6Y45gpjBew4a/JU5A4TFNO8Bq3z",
```

Comment:
I expected this sha to need an update, but I can't find anything that uses it

Reply:
bazel checks against this sha after downloading the dependency. I think it will also cache based on that sha, so if it's not updated you may get an old artifact.